### DISCUSSION TITLE
Ability to filter time by "brushing" on time series visualization?

### DISCUSSION THREAD

**Discussion by Octochef** [apache/superset#21841] [Ideas]
Hi all, coming from Kibana /Grafana i really miss the ability to filter time by "brushing" (dragging..) on a time based visualization.
This i guess would be similar in concept as an "emit filter". 

Is there any reason why this has not been done / asked for in Superset? 

Would anyone of the expert estimate be able to give an even so rough stimate of the complexity of such implementation was one to sponsor such feature? Thanks in advance!

**Comment by orazs** [comment]
3 years later.. any updates on this?
